features/cli/display_failing_result.feature

```diff
@@ -6,5 +6,18 @@ Feature: Display failing result
     Then it should fail with:
       """
       ✗ http://localhost:54321/missing_header.html
-        - Found 0 h1 elements.
+        * Headings > Exactly one main heading
+          - Found 0 h1 elements.
+      """
+
+  Scenario: Two standards fail
+    Given a website running at http://localhost:54321
+    When I run `a11y http://localhost:54321/two_headings_failures.html`
+    Then it should fail with:
+      """
+      ✗ http://localhost:54321/two_headings_failures.html
+        * Headings > Content must follow headings
```

Comment:
I like it!

Would it be picky to suggest using a colon rather than an angle-bracket?

i.e.

`Headings: Content must follow headings`